plugins/outputs/sql/sample.conf

```diff
@@ -18,6 +18,7 @@
   ##  {TABLE} - table name as a quoted identifier
   ##  {TABLELITERAL} - table name as a quoted string literal
   ##  {COLUMNS} - column definitions (list of quoted identifiers and types)
+  ##  {TAG_COLUMNS} - tag column definitions (list of quoted identifiers)
```

Comment:
Please remove this from this PR as this is an unrelated feature!

Reply:
Moving this to a separate PR.

Please note that without this fix, the SQL output plugin simply does not work with the recommended ClickHouse config, neither with v1 nor with v2. The only reason the existing test doesn't catch this is because the test overrides the create table template.

Reply:
Here is the PR: https://github.com/influxdata/telegraf/pull/16464